api/custom-map-ui/geoname-search-ui.html

```diff
@@ -0,0 +1,329 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GeoName Search</title>
+    
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ" crossorigin="anonymous"></script>
+
+    <!--<script type="module" src="Web-Map-Custom-Element-0.8.0/mapml-viewer.js"></script>-->
+
+    <script type="module" src="../../dist/mapml-viewer.js"></script>
+    <style>
+        html, body {
+        height: 100%; /* These styles are required if you wish to use a % based
+                         height value on the mapml-viewer element. */
+        }
+        mapml-viewer:defined {
+            width: 75%;
+            height: 90%;
+        }
+        h2 {
+            width: 100%;
+            text-align: center;
+        }
+        .result {
+            width: 100%;
+            background-color: #78bbc0;
+            color: white;
+            padding: 0.5em 1em;
+            text-decoration: none;
+        }
+        .result:hover {
+            background-color: #555;
+        }
+
+        .result:active {
+            background-color: black;
+        }
+
+        .result:visited {
+            background-color: #ccc;
+        }
+        .results_container {
+            overflow: scroll;
+            display:none;
+            height:100%;
+        }
+        .result_description {
+            font-size: 11px;
+        }
+        .search {
+            height:81%;
```

Comment:
```suggestion
            height: auto;
```
(only take up as much space as needed, currently there's a large gap between the searchbox and the map on smaller screen sizes)